src/harmony/Dropdown/Dropdown.tsx

```diff
@@ -84,6 +85,7 @@ export const DropdownTrigger = ({
     className,
     onClick,
     renderTrigger,
+    arrow = true,
```

Comment:
I don't know how many times we need to talk about default true values. Are u sure u r doing it here for backward compatibility? For the new components w/o any version?